src/test/java/org/fogbowcloud/manager/core/models/linkedlist/SynchronizedDoublyLinkedListTest.java

```diff
@@ -0,0 +1,242 @@
+package org.fogbowcloud.manager.core.models.linkedlist;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import org.fogbowcloud.manager.core.models.orders.ComputeOrder;
+import org.fogbowcloud.manager.core.models.orders.Order;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SynchronizedDoublyLinkedListTest {
+
+	private SynchronizedDoublyLinkedList list;
+	
+	@Before
+	public void initialize() {
+		this.list = new SynchronizedDoublyLinkedList();
+	}
+	
+	@Test
+	public void testAddFirst() {
+		assertNull(this.list.getHead());
+		assertNull(this.list.getTail());
+		
+		Order order = createOrder("0");
+		this.list.addItem(order);
+
+		assertEquals(order, this.list.getHead().getOrder());
+		assertEquals(order, this.list.getCurrent().getOrder());
+		assertEquals(order, this.list.getTail().getOrder());
+	}
+	
+	@Test
+	public void testAddOrder() {
+		assertNull(this.list.getNext());
```

Comment:
What's the purpose of this assert? If it is to guarantee that the list is empty, it would be better to assertNull(this.list.getHead()) which is the definition of an empty list.